### ISSUE TITLE
Support Relay modern template strings

### ISSUE THREAD

**Issue by ajhyndman** [kumarharsh/graphql-for-vscode#35]
Currently syntax highlighting is supported for Relay classic template strings.

```js
gql`
query {
  field
}
`

Relay.QL`
query {
  field
}
`
```

But not Relay modern template strings.

```js
graphql`
query {
  field
}
`
```

It would be great to update this!

**Comment by kumarharsh** [comment]
Fixed in ebfbce3

**Comment by ajhyndman** [comment]
Awesome!  Thanks for a great plugin!
